go/vt/vtgate/planbuilder/testdata/aggr_cases.json

```diff
@@ -2445,10 +2445,10 @@
                       {
                         "OperatorType": "Projection",
                         "Expressions": [
-                          "[COLUMN 0] * [COLUMN 1] as count(*)",
-                          "[COLUMN 2] * [COLUMN 1] as count(ue.foo)",
-                          "[COLUMN 3] as bar",
-                          "[COLUMN 4] as weight_string(us.bar)"
+                          "count(*) * count(*) as count(*)",
+                          "count(ue.foo) * count(*) as count(ue.foo)",
+                          ":3 as bar",
```

Comment:
not clear to me why some offsets are serialized as column names and some as offsets

Reply:
That's up to the planner: if you passed the original column `sqlparser.Expr` when translating the expression, we'll keep it when serializing. Otherwise, we'll print an offset.